1-Draft/RFC00XX-Native-Command-Error-Handling.md

```diff
@@ -0,0 +1,140 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+Area: Core
+Comments Due: 10/31/2020
+---
+
+# Native Command Error Handling
+
+PowerShell scripts using native commands would benefit from being able to use error handling
+features like those used by cmdlets.
+
+## Motivation
+
+In PowerShell by default, script processing continues when non-terminating errors occur. This is a
+benefit when expecting non-terminating errors in normal execution such as non-responsive computers
+from a list. This default behavior is controlled with the preference variable
+`$ErrorActionPreference` default of `Continue`.
+
+In production, often customers prefer that script execution stops when a non-terminating error
+occurs. This is particularly true in CI where the preference is to fail fast. PowerShell currently
+supports customers with this ability by setting the `$ErrorActionPreference` variable in the script
+to `Stop`.
+
+```Powershell
+$ErrorActionPreference = 'Stop'
+```
+
+Native commands usually return an exit code to the calling application which will be zero for
+success or non-zero for failure. However, native commands currently do not participate in the
+PowerShell error stream. Redirected `stderr` output is not interpreted the same as the PowerShell
```

Comment:
stderr not being interpreted as an error by $ErrorActionPreference was a new experimental feature added to 7.1